### ISSUE TITLE
Leverage Auto-Scaling API

### ISSUE THREAD

**Issue by jakepeyser** [IBM-Bluemix/auto-scaling-demo#11]
Begin using the auto-scaling API to programmatically create a policy. Refactor the code to incorporate the setting of policy stats in the UI

**Comment by jthomas** [comment]
There's an API for auto-scaling now?

**Comment by jakepeyser** [comment]
@jthomas yep, [here are the Swagger API docs](https://www.ng.bluemix.net/docs/api/content/api/auto-scaling/index.html#/)
